roles/ceph-common/tasks/checks/check_ntp_atomic.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: check ntp installation on atomic
+  command: rpm -q chrony
+  register: ntp_pkg_query
+  ignore_errors: true
+  changed_when: false
+  when: ansible_os_family == 'RedHat'
```

Comment:
The `when` is a bit redundant with the `when` in the include.
Moreover, shouldn't we check for Atomic specifically?